pkg/servicemesh/federation/model/model.go

```diff
@@ -23,9 +23,8 @@ type ServiceKey struct {
 }
 
 type ServiceListMessage struct {
-	Checksum                uint64             `json:"checksum" hash:"ignore"`
-	NetworkGatewayEndpoints []*ServiceEndpoint `json:"networkGatewayEndpoints,omitempty" hash:"set"`
```

Comment:
Will removing this field cause an incompatibility between different releases? (Is this message passed from one cluster to another? If they run different versions of istiod, will this change cause issues?)

Reply:
nope it won't cause issues. as I mentioned, we don't need the endpoints that come over the wire to establish connections, and this is all transmitted in JSON, so if a field is not there it'll just be `nil` on the other side. The only place we're using it is in the loop where we `range` over it, and that will work

Reply:
ok, thanks